src/site/includes/header.html

```diff
@@ -71,6 +71,9 @@
   <script defer data-entry-name="vendor.js"></script>
   <script defer data-entry-name="{{ entryname | default: 'static-pages' }}.js"></script>
 
+  <!-- Mapbox map -->
+  <link href='https://api.mapbox.com/mapbox-gl-js/v1.12.0/mapbox-gl.css' rel='stylesheet' />
```

Comment:
- This will add this css to every page on the site. Does this map show up on every page on the site?
- The preferred way is to import CSS as a bundle instead of using a third party CDN to host it. Doesn't look like mapbox [documents](https://docs.mapbox.com/mapbox-gl-js/api/) how to do this so I think the options are 
  - use the CDN hosted CSS only on the pages that the map shows up on 
  - try to download the css and bundle it as part of your app